smartlearner/tasks.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, msg, *views, **recurrent_options):
 
     def execute(self, status):
         values = [view.view(status) for view in self.views]
-        print(self.msg.format(*values))
+        print("{}:{}  ".format(status.current_epoch, status.current_update_in_epoch), self.msg.format(*values))
```

Comment:
It's is a good idea and should be the default, but we should make it optional. It might happens we don't want to print the epoch number.